### PR TITLE
Tabs and language Changer are now in a new section so that the language Changer can be used without Tabs.

### PR DESCRIPTION
**Pull request by ReneZumtobel**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | yes
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | fixes #issuenum
| Related issues/PRs | https://github.com/sulu/sulu/pull/2821
| License | MIT
| Documentation PR | sulu/sulu-docs#prnum

#### What's in this PR?

Tabs and the language Changer are now in new subheader sector and now you can use a language changer without a tab menu.

#### Why?

Which problem does the PR fix? (remove this section if you linked an issue above)

### CODE REVIEW

**Comment by ReneZumtobel** [comment]
@marcelmoosbrugger, @danrot

File: husky_components/overlay/main.js [hunk 1]
```diff
@@ -682,6 +682,11 @@ define([], function() {
             this.overlay.slides[slide].$content = this.sandbox.dom.find(constants.contentSelector, this.overlay.slides[slide].$el);
             this.overlay.slides[slide].$header = this.sandbox.dom.find(constants.headerSelector, this.overlay.slides[slide].$el);
 
+            if(this.slides[slide].languageChanger !== null || this.slides[slide].tabs !== null){
```
Comment (marcelmoosbrugger) [inline husky_components/overlay/main.js]: There should be spaces between "if" and "(" als well as ")" and "{"

File: husky_components/overlay/main.js [hunk 2]
```diff
@@ -682,6 +682,11 @@ define([], function() {
...
+                this.overlay.slides[slide].$subheader = this.sandbox.dom.createElement('<div class="' + constants.subheaderClass + '"/>');
+                this.sandbox.dom.after(this.overlay.slides[slide].$header, this.overlay.slides[slide].$subheader);
```
Comment (marcelmoosbrugger) [inline husky_components/overlay/main.js]: The jQuery abstraction should not be done anymore. The new code we write is allowed to directly call the jQuery functions. In this example:
`this.overlay.slides[slide].$header.after(this.overlay.slides[slide].$subheader)`

File: husky_components/overlay/main.js [hunk 3]
```diff
@@ -702,9 +707,10 @@ define([], function() {
 
             this.sandbox.dom.addClass(this.overlay.$el, 'has-language-chooser');
             this.overlay.slides[slide].$languageChanger = this.sandbox.dom.createElement(
-                '<div class="' + constants.languageChangerClass + '"/>'
+                this.slides[slide].tabs !== null?'<div class="' + constants.languageChangerClass + '"/>' :
```
Comment (marcelmoosbrugger) [inline husky_components/overlay/main.js]: Spaces around the "?"

File: husky_components/overlay/main.js [hunk 4]
```diff
@@ -702,9 +707,10 @@ define([], function() {
...
+                '<div style="height: 51px;"/> <div class="' + constants.languageChangerClass + '-frame"/>'
```
Comment (marcelmoosbrugger) [inline husky_components/overlay/main.js]: Inline styles are generally considered bad practice. Try to use another mechanism to set your styles. Preferably with CSS.

File: husky_components/overlay/main.js [hunk 5]
```diff
@@ -702,9 +707,10 @@ define([], function() {
...
             );
-            this.sandbox.dom.append(this.overlay.slides[slide].$content, this.overlay.slides[slide].$languageChanger);
+            this.sandbox.dom.append(this.overlay.slides[slide].$subheader, this.overlay.slides[slide].$languageChanger);
```
Comment (marcelmoosbrugger) [inline husky_components/overlay/main.js]: Native jQuery functions instead of abstraction.

File: husky_components/overlay/main.js [hunk 6]
```diff
@@ -813,7 +817,7 @@ define([], function() {
          */
         startTabsComponent: function(slide) {
             var $element = this.sandbox.dom.createElement('<div/>');
-            this.sandbox.dom.html(this.overlay.slides[slide].$tabs, $element);
+            this.sandbox.dom.append(this.overlay.slides[slide].$subheader, $element);
```
Comment (marcelmoosbrugger) [inline husky_components/overlay/main.js]: Native jQuery functions instead of abstraction.

File: scss/modules/overlay.scss [hunk 7]
```diff
@@ -115,10 +115,25 @@
...
+
+    .language-changer {
+        width: 80px;
+        margin: 10px 0;
+        top: 0;
+        position: absolute;
+        right: 20px;
+
+        &-frame{
+            @extend .language-changer;
```
Comment (marcelmoosbrugger) [inline scss/modules/overlay.scss]: Why does the frame extend the language-changer. Does the sentence "The language-changer frame is a language-changer" make sense? If not the mutal styles should not be set with extend but maybe with:
```
.language-changer,
.language-changer-frame {...}
```

**Comment by marcelmoosbrugger** [comment]
@danrot This PR is needed for the planned changes regarding media-cropping (a dropdown needs to be added to the header of an overlay). Do you have any plans on merging this PR?

**Pull request closed without merging**